include/SANDDigitizationEDEPSIM.h

```diff
@@ -48,12 +48,34 @@ void group_hits_by_tube(TG4Event* ev, const SANDGeoManager& geo,
 
 void create_digits_from_hits(const SANDGeoManager& geo,
                              std::map<int, std::vector<hit> >& hits2Tube,
-                             std::vector<dg_tube>& digit_vec);
+                             std::vector<dg_wire>& digit_vec);
 
 void digitize_stt(TG4Event* ev, const SANDGeoManager& geo,
-                  std::vector<dg_tube>& digit_vec);
+                  std::vector<dg_wire>& digit_vec);
 }  // namespace stt
 
+namespace chamber
+{
+void group_hits_by_wire(TG4Event* ev, const SANDGeoManager& geo,
+                        std::map<int, std::vector<hit> >& hits2wire);
+
+bool isInWire(SANDWireInfo& wire, TVector3& point);
+
+bool isInHit(hit& h, TVector3& point);
+
+std::vector<TLorentzVector> WireHitClosestPoints(hit& h,
+                                                 SANDWireInfo& arg_wire);
+
+double GetMinWireTime(TLorentzVector point, SANDWireInfo& arg_wire);
+
+void create_digits_from_wire_hits(const SANDGeoManager& geo,
+                                  std::map<int, std::vector<hit> >& hits2wire,
```

Comment:
it looks like the map is not modified by the function, can we pass it as a `const&` ?